app/src/main/java/org/mozilla/focus/browser/BrowserFragment.kt

```diff
@@ -117,6 +120,11 @@ class BrowserFragment : IWebViewLifecycleFragment() {
         })
     }
 
+    private fun initMediaSession() {
+        videoVoiceCommandMediaSession = VideoVoiceCommandMediaSession(activity!!)
```

Comment:
Are `!!` a case to use NonFatalAssertionException? Not saying you need to, just wondering.

Reply:
The activity is guaranteed to be attached here: the lifecycle is `onAttach`, `onCreate`.